### DISCUSSION TITLE
Add translation  service support for the AI assistant Kimi, produced by Moonshot AI.

### DISCUSSION THREAD

**Discussion by vizmoe** [immersive-translate/immersive-translate#1647] [Ideas]
### 新功能 | New Feature

1. Add translation  service support for the AI assistant Kimi, produced by Moonshot AI.

### 为什么您想要添加这些功能 | Why you want to add these features

1. Users in mainland China can access the Kimi large language model more conveniently without the need for a VPN.
2. Kimi demonstrates high response speed and translation quality.

### 提议 | Proposal

The openai-translator browser extension now supports Kimi as a translation service, see https://github.com/openai-translator/openai-translator/pull/1427.

**Comment by theowenyoung** [comment]
For users, considering the large number of API calls for Immersive Translate, accessing monnshot AI in this way has great risks for the user's account, and the user's account is likely to be banned due to such calls. We are considering accessing moonshot's API, but the rpm provided by their API is only 3, which is far from the frequency of calls for immersive translation.
